docs/build/local-data-storage.md

```diff
@@ -6,60 +6,85 @@ A peer hosting services may agree to provide access to their filesystem or alloc
 
 While Wasm modules in general are sandboxed to not have access to host resources, Marine Wasm modules may be granted access to a variety of host resources iff the host peer agrees to such access. The request for resource access and allocation come in form of deployment parameters, which we'll see very soon
 
-The [accompanying repo](notion://www.notion.so/fluencenetwork/Fluence-Developer-Documentation-bdf8d06ad52e493fb765456dbd5480cd).
-
 Let's start by creating a new, minimal Fluence CLI project by the name of your choice:
 
-```
+```bash
 fluence init write-to-file
+```
+```
+vasd85@MacBook-Pro test % fluence init write-to-file
```

Comment:
I would suggest to remove your personal terminal prefix from docs, just replace it with $